run-ecs-task/action.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# run task from task definition
+task_run=$(aws ecs run-task --task-definition "${ECS_TASK_DEFINITION}" --cluster "${ECS_CLUSTER}" --launch-type FARGATE --network-configuration "${ECS_TASK_NETWORK_CONFIGURATION}" --overrides "${ECS_TASK_OVERRIDES}")
```

Comment:
Will this still work as expected if the inputs populating `ECS_TASK_NETWORK_CONFIGURATION` and `ECS_TASK_OVERRIDES` are undefined?

Reply:
Will check. It was working for my use case, but it's a different setup from MBTA's infrastructure.